providers/implementations/rands/seeding/rand_unix.c

```diff
@@ -76,7 +76,7 @@ static uint64_t get_timer_bits(void);
  * macro that might be undefined.
  */
 # undef OSSL_POSIX_TIMER_OKAY
-# if defined(_POSIX_TIMERS) && _POSIX_TIMERS > 0
+# if defined(_POSIX_TIMERS) && _POSIX_TIMERS - 0 > 0
```

Comment:
I wonder if there are preprocessors that will object to separating `-` and `0` with a space?

Reply:
I think a comment here explaining why we do "-0" here would be helpful. It is not immediately obvious and I could imagine someone "helpfully" cleanly this "deadcode" away in the future.

I would also remove the spaces are "-". Not our normal coding style, but justifiable here I think.

Reply:
Okay, I've removed the space and added a comment.